src/builders.ts

```diff
@@ -103,7 +106,7 @@ export function makeSTXTokenTransfer(
   const signer = new TransactionSigner(transaction);
   signer.signOrigin(privKey);
 
-  return transaction;
+  return Promise.resolve(transaction);
```

Comment:
`Promise.resolve` is implicit when just returning from an `async` fn right? Is the resolve just because there's no `await` yet?